django-stubs/core/handlers/base.pyi

```diff
@@ -21,3 +24,5 @@ class BaseHandler:
     def check_response(self, response: HttpResponseBase, callback: Any, name: str | None = ...) -> None: ...
     def make_view_atomic(self, view: Callable[..., HttpResponseBase]) -> Callable[..., HttpResponseBase]: ...
     def process_exception_by_middleware(self, exception: Exception, request: HttpRequest) -> HttpResponse: ...
+
+def reset_urlconf(sender: object, **kwargs: Any) -> None: ...
```

Comment:
```suggestion
import _typeshed

def reset_urlconf(sender: _typeshed.Unused, **kwargs: Any) -> None: ...
```

Reply:
Thanks! I didn't know that feature :) dc3b266